src/utils/viewer/viewer.cpp

```diff
@@ -466,14 +466,20 @@ Mn::Vector3 Viewer::positionOnSphere(Mn::SceneGraph::Camera3D& camera,
   return (result * Mn::Vector3::yScale(-1.0f)).normalized();
 }
 
+float timeSinceLastSimulation = 0.0;
 void Viewer::drawEvent() {
   Mn::GL::defaultFramebuffer.clear(Mn::GL::FramebufferClear::Color |
                                    Mn::GL::FramebufferClear::Depth);
   if (sceneID_.size() <= 0)
     return;
 
   if (physicsManager_ != nullptr)
-    physicsManager_->stepPhysics(timeline_.previousFrameDuration());
+    // step physics at a fixed rate
+    timeSinceLastSimulation += timeline_.previousFrameDuration();
+  if (timeSinceLastSimulation > 1.0 / 60.0) {
+    physicsManager_->stepPhysics(1.0 / 60.0);
+    timeSinceLastSimulation = 0.0;
+  }
```

Comment:
What if timeSinceLastSimulation == 1.0/60.0?  Will this be a problem - would you not want to step again, or is that already accounted for?